execution/gethexec/sequencer.go

```diff
@@ -710,7 +723,116 @@ func (s *Sequencer) precheckNonces(queueItems []txQueueItem) []txQueueItem {
 	return outputQueueItems
 }
 
+func (s *Sequencer) createBlockEspresso(ctx context.Context) (returnValue bool) {
```

Comment:
I think we should move these code to the `arbnode` to let every validator forward txs and build block. `arb sequencer` is a centralized thing

Reply:
And it is not easy to update `arb sequencer` to let every one run it because arbitrum now is not [permissionless](https://docs.arbitrum.foundation/state-of-progressive-decentralization).